pkg/profile/artifact_test.go

```diff
@@ -130,276 +123,143 @@ var _ = Describe("Profile", func() {
-				Expect(helmRelease.Spec.Chart.Spec.SourceRef).To(Equal(
-					helmv2.CrossNamespaceObjectReference{
-						Kind:      helmRepoKind,
-						Name:      helmRefName,
-						Namespace: namespace,
-					},
-				))
-				Expect(helmRelease.GetValues()).To(Equal(map[string]interface{}{
-					"replicaCount": float64(3),
-					"service": map[string]interface{}{
-						"port": float64(8081),
-					},
-				}))
-				Expect(helmRelease.Spec.ValuesFrom).To(Equal([]helmv2.ValuesReference{
-					{
-						Name:     "nginx-values",
-						Kind:     "Secret",
-						Optional: true,
-					},
-				}))
-			})
-		})
-
-		When("the branch name for a git repository is not domain compatible", func() {
-			It("will sanitise it", func() {
-				pSub = profilesv1.ProfileInstallation{
-					TypeMeta: profileTypeMeta,
-					ObjectMeta: metav1.ObjectMeta{
-						Name:      installationName,
-						Namespace: namespace,
-					},
-					Spec: profilesv1.ProfileInstallationSpec{
-						Source: &profilesv1.Source{
-							URL:    profileURL,
-							Branch: "not_domain_compatible",
-						},
-					},
-				}
-				maker := profile.NewProfilesArtifactsMaker(profile.MakerConfig{
-					GitClient:        fakeGitClient,
-					RootDir:          rootDir,
-					GitRepoNamespace: gitRepoNamespace,
-					GitRepoName:      gitRepoName,
-				})
-				artifacts, err := maker.MakeArtifacts(pSub)
-				Expect(err).NotTo(HaveOccurred())
-				Expect(artifacts).To(HaveLen(4))
-
-				By("generating the path based helm release artifact", func() {
-					pathBasedHelmArtifact := artifacts[1]
-					Expect(pathBasedHelmArtifact.Name).To(Equal(chartName2))
-
-					objects := pathBasedHelmArtifact.Objects
-
-					helmReleaseName := fmt.Sprintf("%s-%s-%s", installationName, profileName1, chartName2)
-					helmRelease := objects[0].(*helmv2.HelmRelease)
-					Expect(helmRelease.Name).To(Equal(helmReleaseName))
-					Expect(err).NotTo(HaveOccurred())
-					Expect(helmRelease.Spec.Chart.Spec.Chart).To(Equal("root-dir/artifacts/chartTwoArtifactName/chart/artifact/path-two"))
-					Expect(helmRelease.Spec.Chart.Spec.SourceRef).To(Equal(
-						helmv2.CrossNamespaceObjectReference{
-							Kind:      gitRepoKind,
-							Name:      gitRepoName,
-							Namespace: gitRepoNamespace,
-						},
-					))
-				})
+			By("generating the path based kustomization artifact", func() {
+				content, err := ioutil.ReadFile(files["Kustomization.yaml"])
+				Expect(err).ToNot(HaveOccurred())
+				Expect(string(content)).To(Equal(fmt.Sprintf(`apiVersion: kustomize.toolkit.fluxcd.io/v1beta1
+kind: Kustomization
+metadata:
+  creationTimestamp: null
+  name: mySub-weaveworks-nginx-nginx-deployment
+  namespace: default
+spec:
+  interval: 5m0s
+  path: %s/artifacts/nginx-deployment/nginx/deployment
+  prune: true
+  sourceRef:
+    kind: GitRepository
+    name: git-repo-name
+    namespace: git-repo-namespace
+  targetNamespace: default
+status: {}
+`, rootDir)))
```

Comment:
we need to test all the other artifacts are correct too

Reply:
What do you mean? Like the others a couple lines below this one?

Reply:
And the profile-installation. What... the... heck. I thought I'm testing that.

Reply:
Added all the things!

Reply:
:smile: